2021-12-01 20:08:31,523 INFO    MainThread:15749 [wandb_setup.py:_flush():71] setting env: {}
2021-12-01 20:08:31,523 INFO    MainThread:15749 [wandb_setup.py:_flush():71] setting login settings: {}
2021-12-01 20:08:31,523 INFO    MainThread:15749 [wandb_setup.py:_flush():71] setting login settings: {}
2021-12-01 20:08:31,523 INFO    MainThread:15749 [wandb_init.py:_log_setup():371] Logging user logs to /geoinfo_vol1/home/z/h/zhao2/vit-keras/wandb/run-20211201_200831-smq88svr/logs/debug.log
2021-12-01 20:08:31,524 INFO    MainThread:15749 [wandb_init.py:_log_setup():372] Logging internal logs to /geoinfo_vol1/home/z/h/zhao2/vit-keras/wandb/run-20211201_200831-smq88svr/logs/debug-internal.log
2021-12-01 20:08:31,524 INFO    MainThread:15749 [wandb_init.py:init():404] calling init triggers
2021-12-01 20:08:31,524 INFO    MainThread:15749 [wandb_init.py:init():411] wandb.init called with sweep_config: {}
config: {}
2021-12-01 20:08:31,525 INFO    MainThread:15749 [wandb_init.py:init():449] starting backend
2021-12-01 20:08:31,525 INFO    MainThread:15749 [backend.py:_multiprocessing_setup():97] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-12-01 20:08:31,549 INFO    MainThread:15749 [backend.py:ensure_launched():199] starting backend process...
2021-12-01 20:08:31,565 INFO    MainThread:15749 [backend.py:ensure_launched():205] started backend process with pid: 15884
2021-12-01 20:08:31,567 INFO    MainThread:15749 [wandb_init.py:init():458] backend started and connected
2021-12-01 20:08:31,570 INFO    MainThread:15749 [wandb_init.py:init():519] updated telemetry
2021-12-01 20:08:31,621 INFO    MainThread:15749 [wandb_init.py:init():549] communicating current version
2021-12-01 20:08:32,973 INFO    MainThread:15749 [wandb_init.py:init():554] got version response 
2021-12-01 20:08:32,973 INFO    MainThread:15749 [wandb_init.py:init():564] communicating run to backend with 30 second timeout
2021-12-01 20:08:33,232 INFO    MainThread:15749 [wandb_init.py:init():592] starting run threads in backend
2021-12-01 20:08:38,225 INFO    MainThread:15749 [wandb_run.py:_console_start():1816] atexit reg
2021-12-01 20:08:38,227 INFO    MainThread:15749 [wandb_run.py:_redirect():1690] redirect: SettingsConsole.REDIRECT
2021-12-01 20:08:38,229 INFO    MainThread:15749 [wandb_run.py:_redirect():1695] Redirecting console.
2021-12-01 20:08:38,233 INFO    MainThread:15749 [wandb_run.py:_redirect():1751] Redirects installed.
2021-12-01 20:08:38,233 INFO    MainThread:15749 [wandb_init.py:init():619] run started, returning control to user process
2021-12-01 20:08:51,227 INFO    MainThread:15749 [wandb_run.py:_atexit_cleanup():1786] got exitcode: 1
2021-12-01 20:08:51,228 INFO    MainThread:15749 [wandb_run.py:_restore():1758] restore
2021-12-01 20:08:53,751 INFO    MainThread:15749 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 855
  total_bytes: 855
}

2021-12-01 20:08:53,941 INFO    MainThread:15749 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 855
  total_bytes: 855
}

2021-12-01 20:08:54,207 INFO    MainThread:15749 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 855
  total_bytes: 855
}

2021-12-01 20:08:54,323 INFO    MainThread:15749 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 855
  total_bytes: 16619
}

2021-12-01 20:08:54,425 INFO    MainThread:15749 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 855
  total_bytes: 16619
}

2021-12-01 20:08:54,527 INFO    MainThread:15749 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 855
  total_bytes: 16619
}

2021-12-01 20:08:54,628 INFO    MainThread:15749 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 855
  total_bytes: 16619
}

2021-12-01 20:08:54,730 INFO    MainThread:15749 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 855
  total_bytes: 16619
}

2021-12-01 20:08:54,833 INFO    MainThread:15749 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 855
  total_bytes: 16619
}

2021-12-01 20:08:54,935 INFO    MainThread:15749 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 855
  total_bytes: 16619
}

2021-12-01 20:08:55,037 INFO    MainThread:15749 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 855
  total_bytes: 16619
}

2021-12-01 20:08:55,139 INFO    MainThread:15749 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 855
  total_bytes: 16619
}

2021-12-01 20:08:55,241 INFO    MainThread:15749 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 855
  total_bytes: 16619
}

2021-12-01 20:08:55,342 INFO    MainThread:15749 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 855
  total_bytes: 16619
}

2021-12-01 20:08:55,443 INFO    MainThread:15749 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 855
  total_bytes: 16619
}

2021-12-01 20:08:55,545 INFO    MainThread:15749 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 855
  total_bytes: 16619
}

2021-12-01 20:08:55,646 INFO    MainThread:15749 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 855
  total_bytes: 16619
}

2021-12-01 20:08:55,750 INFO    MainThread:15749 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 5665
  total_bytes: 16619
}

2021-12-01 20:08:55,852 INFO    MainThread:15749 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 16619
  total_bytes: 16619
}

2021-12-01 20:08:55,953 INFO    MainThread:15749 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 16619
  total_bytes: 16619
}

2021-12-01 20:08:56,055 INFO    MainThread:15749 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 16619
  total_bytes: 16619
}

2021-12-01 20:08:56,156 INFO    MainThread:15749 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 16619
  total_bytes: 16619
}

2021-12-01 20:08:56,258 INFO    MainThread:15749 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 16619
  total_bytes: 16619
}

2021-12-01 20:08:56,488 INFO    MainThread:15749 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 16619
  total_bytes: 16619
}

2021-12-01 20:08:57,018 INFO    MainThread:15749 [wandb_run.py:_wait_for_finish():1916] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 16619
  total_bytes: 16619
}
local_info {
}

2021-12-01 20:08:58,337 INFO    MainThread:15749 [wandb_run.py:_append_files():2184] logging synced files
